tasks/utils.wdl

```diff
@@ -0,0 +1,50 @@
+version development
+
+'''
+additional useful tasks
+'''
+
+task GetGeneChrPairs {
+
+    input {
+        File featureVariantFile
+        Array[String]+ columnsToSelect # chrom, gene
+    }
+
+	command <<<
+
+cat << EOF > script.py
+import csv
+
+columns_to_select = '~{sep(",", columnsToSelect)}'.split(",")
+# we expect output PAIRS, so make sure there are 2 columns to select
+assert len(columns_to_select) == 2
+with open("~{featureVariantFile}") as f, open("outputPairs.tsv", "w+") as w:
+    reader = csv.reader(f)
+
+    # get headers list to get a header to indices 
+    headers = next(reader)
+    # this will fail if column isn't in the header
+    indices_to_select = [headers.index(h) for h in columns_to_select]
+
+    cut_lines = [[l[i] for i in indices_to_select] for l in reader]
+    unique_lines = set('\t'.join(l) for l in cut_lines)
+    if unique_lines:
+    # this should leave an empty fails
+        w.write("\n".join(sorted(unique_lines)))
+
+EOF
+
+python script.py
+
+	>>>
+
+    runtime {
+        memory: "2G"
+		container: "python:3.10"
```

Comment:
Fix the indentation here

Reply:
This is probably awkward to see, but the first line uses spaces, the second uses tabs.